code-challenges.js

```diff
@@ -13,52 +13,92 @@ var num3 = -7
 // Expected output: "-7 is not divisble by three"
 
 
+const divisible = (num) => {
+    if(num % 3 === 0){
+    return num + " is divisible by three"
+} else if (num % 3 >= 1){
+    return num + " is not divisible by three"
+} else if (num % 3 >= -3){
+    return num + " is not divisible by three"
+}else{
+    return "must type in a number"
+}
+}
+console.log(divisible(num1))
+console.log(divisible(num2))
+console.log(divisible(num3))
 
 
 
-// --------------------2) Create a function that takes in the randomNouns variable and returns an array with all the words capitalized.
+// // --------------------2) Create a function that takes in the randomNouns variable and returns an array with all the words capitalized.
 
 var randomNouns = ["streetlamp", "potato", "teeth", "conclusion", "nephew", "temperature", "database"]
-// Expected output: ["Streetlamp", "Potato", "Teeth", "Conclusion", "Nephew", "Temperature", "Database"]
+// // Expected output: ["Streetlamp", "Potato", "Teeth", "Conclusion", "Nephew", "Temperature", "Database"]
+const capitalizer = (array) => {
+    let eachItem = array.map(value => {
+        return value[0].toUpperCase() + value.substring(1)
+    })
+    return eachItem.join(", ")
+}
+console.log(capitalizer(randomNouns))
 
 
-
-
-
-// --------------------3) Create a function that takes in the mixedDataArray array and returns an array with ONLY NUMBERS sorted from least to greatest.
+// // --------------------3) Create a function that takes in the mixedDataArray array and returns an array with ONLY NUMBERS sorted from least to greatest.
 
 var mixedDataArray = [true, 8, "hello", 90, -8, null, 0, 46, 59, 107, "hey!"]
-// Expected output: [-8, 0, 8, 46, 59, 90, 107]
+// // Expected output: [-8, 0, 8, 46, 59, 90, 107]
 
+const onlyNum = mixedDataArray.filter(function (item) {
+    return (parseInt(item) == item);
+ 
+});
 
+console.log(onlyNum.sort((a, b) => a - b))
```

Comment:
great! the logic is all here, just think about how you might add the `.sort` functionality into your actual function in order for it to be as reusable as possible!